src/everest/detached/jobs/everserver.py

```diff
@@ -273,7 +274,8 @@ def get_shared_data(
 
 
 def _find_open_port(host, lower, upper) -> int:
-    for port in range(lower, upper):
+    for _ in range(5):
+        port = random.randint(lower, upper)
```

Comment:
Maybe add a comment just to have an idea why this was needed so that once we update further everserver we might undo it.